flash/video/classification/utils.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, Callable, List, Optional, Tuple, Type
+
+import torch
+
+from flash.core.utilities.imports import _VIDEO_AVAILABLE
+
+if _VIDEO_AVAILABLE:
+    from pytorchvideo.data.utils import MultiProcessSampler
+else:
+    MultiProcessSampler = None
+
+
+class LabeledVideoTensorDataset(torch.utils.data.IterableDataset):
+    """LabeledVideoTensorDataset handles a direct tensor input data."""
+
+    def __init__(
+        self,
+        labeled_video_tensors: List[Tuple[str, Optional[dict]]],
+        video_sampler: Type[torch.utils.data.Sampler] = torch.utils.data.RandomSampler,
+        transform: Optional[Callable[[dict], Any]] = None,
```

Comment:
As we don't use `transform` it may be cleaner to remove it

Reply:
Done! :)